articles/azure-sql-edge/date-bucket-tsql.md

```diff
@@ -89,33 +89,33 @@ The return value data type for this method is dynamic. The return type depends o
 `Date_Bucket` returns the latest date or time value, corresponding to the datePart and number parameter. For example, in the expressions below, `Date_Bucket` will return the output value of `2020-04-13 00:00:00.0000000`, as the output is calculated based on one week buckets from the default origin time of `1900-01-01 00:00:00.000`. The value `2020-04-13 00:00:00.0000000` is 6276 weeks from the origin value of `1900-01-01 00:00:00.000`.
```

Comment:
I missed one `datePart` here - not sure why the `P` is capitalized all over the place.